libraries/AP_MSP/AP_MSP_Telem_Backend.cpp

```diff
@@ -584,9 +591,9 @@ MSPCommandResult AP_MSP_Telem_Backend::msp_process_out_api_version(sbuf_t *dst)
         uint8_t major;
         uint8_t minor;
     } api_version  {
-        proto : MSP_PROTOCOL_VERSION,
```

Comment:
Yeah, obviously gratuitous whitespace changes are a no-no :)

Reply:
Trying to fix formatting I ran the astyle tool with the style sheet checked in under tools and submitted that...
So other than that not sure how to resolve the formatting issues...